test/index.js

```diff
@@ -1,6 +0,0 @@
-'use strict';
```

Comment:
What's wrong with linting? We shifted to using eslint for node services, but removing linting completely seems a bit extreme.

Reply:
We do linting differently. There's another PR ( #44 ) about it and I didn't want to get into that in this PR.

Reply:
Oh, ok, gotcha, I'm out of context here